Screens/ImageCapture.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react'
+import {
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native'
+import Camera from 'react-native-camera'
+
+class ImageCapture extends Component {
+  state = {
+    images: []
+  }
+
+  static navigationOptions = {
+    title: 'Camera'
+  }
+
+  takePicture() {
+    const options = {}
+    this.camera.capture({ metadata: options })
+      .then((data) => {
+        console.log('image captured')
+        console.log('data: ', data)
+        this.props.navigation.state.params.setImages(data)
```

Comment:
>state.params.setImages(data) [](start = 30, length = 28)

This seems like a very strange way to pass images around - can you think of a better way to notify other components of the location of the captured image?

Reply:
That being said, I wouldn't call myself a `react-navigation` expert, so if this is the recommended way to pass a callback to another component, then so be it.